tools/edit.py

```
@@ -99,86 +110,37 @@ async def __call__(
+            # If relative path, make it relative to PROJECT_DIR
+            return PROJECT_DIR / p
         except Exception as e:
-            raise ToolError(f"Invalid path format: {path}. Error: {str(e)}")
-
-        # Check if it's an absolute path
-        if not path.is_absolute():
-            suggested_path = Path.cwd() / path
-            raise ToolError(
-                f"The path {path} is not an absolute path. Maybe you meant {suggested_path}?"
-            )
+            raise ValueError(f"Invalid path: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)